dbc2val/canplayer.py

```diff
@@ -0,0 +1,98 @@
+########################################################################
```

Comment:
Im open for any other names. 
I give this name, because the name in the python spec
https://python-can.readthedocs.io/en/master/scripts.html#can-player